app/models/site_config.rb

```diff
@@ -19,7 +19,7 @@ class SiteConfig < RailsSettings::Base
   # staff account
   field :staff_user_id, type: :integer, default: 1
   field :default_site_email, type: :string, default: "yo@dev.to"
```

Comment:
I'll remove the other defaults in a separate PR. I don't want to clutter this PR.

Reply:
Good idea.